arches_component_lab/src/arches_component_lab/types.ts

```diff
@@ -24,3 +24,12 @@ export interface CardXNodeXWidgetData {
         component: string;
     };
 }
+
+export interface Language {
```

Comment:
this is a good opportunity to flesh out core arches types I think

Reply:
I'll defer to @apeters re: project scope, might be better to do this under lingo.

Reply:
I agree with both of you 😂 .  This should be done but not under modular reports